generate_jlc_files.py

```diff
@@ -79,7 +78,8 @@ def main():
 	if netlist_path is None:
 		logging.error((
 			"Failed to find netlist file: {} in {} (and sub-directories). "
-			"Run 'Tools -> Generate Bill of Materials' in Eeschema (any format). "
+			"It is INPUT_DIRECTORY a KiCad Project? "
```

Comment:
"Is the input directory a KiCad project?"

Reply:
@matthewlai changed.